raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -121,3 +203,49 @@ def test_two_clients_cross_messaging_same_topic(comms_nodes):
         received_message = notification_to_payload(resp)
         assert received_message == payload_2
         break  # only 1 message is expected
+
+
+@pytest.mark.parametrize("amount_of_nodes", [1])
+@pytest.mark.xfail(reason="incorrect exception from comms")
```

Comment:
line `221` is outputting "no peers in routing table" or something of the sort, rather than the "not subscribed" error that we'd expect

Reply:
this should be a UnsubscribeError

can you add this to pending issues?  this is important for exception handling

Reply:
> this should be a UnsubscribeError
> 
> can you add this to pending issues? this is important for exception handling

yes, it's there. i just have to describe it better, but i'm on it